include/CppUTest/CppUTestConfig.h

```diff
@@ -199,6 +199,27 @@
 #define CPPUTEST_CHAR_BIT 8
 #endif
 
+/* Support for "long long" type */
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
Even memcpy wouldn't work as it wouldn't work with literals.

It seems that the only way around this is through fixing the packaging. So, during configure, we'll need to generate a file that specifies CppUTest specific #defines and then install that also. It would need to be empty by default in order not to break the builds of other systems.

I think I'll be looking into this tonight, or did you have happened to do a similar thing yet? We might jsut use CppUTestConfig or need to add CppUTestConfigGenerated.h or so and include it in CppUTestConfig.h